package.json

```diff
@@ -97,6 +97,7 @@
     "hoist-non-react-statics": "^3.0.1",
     "instabug-reactnative": "^9.1.6",
     "io-ts": "1.8.5",
+    "io-react-native-zendesk": "https://github.com/pagopa/io-react-native-zendesk.git#0.3.1",
```

Comment:
Sorry my fault about suggestion
we should match the tag with the version exposed in packages.json
https://github.com/pagopa/io-react-native-zendesk/blob/e2ba121ba5fb6c07c4bfe9748d9685301dd6ca31/package.json#L3
So we should create and refer to the tag `0.3.4`

Reply:
Yes,